app/services/out_of_band_session_accessor.rb

```diff
@@ -35,7 +60,10 @@ def put(data, expiration = 5.minutes)
       send(:set_session, {}, session_uuid, session_data, expire_after: expiration.to_i)
   end
 
-  private
+  # @return [Hash]
+  def load
+    session_store.send(:load_session_from_redis, session_uuid) || {}
```

Comment:
Would it be possible to memoize this? The one section that could benefit is [here](https://github.com/18F/identity-idp/pull/7295/files#diff-828bce10c486ea84e1a09713d23df0dcaa826231e063cd7ef863022a88189174R114-R132) where it potentially loads the session from Redis twice to fetch PII and X509 attributes separately.

Reply:
found another allocation/instantiation we could save in 0c5a41537be9f69508aabef11da360e273313821, I think this will save creating a new redis connection